tokens/style-dictionary.js

```diff
@@ -338,13 +338,20 @@ const initializeStyleDictionary = async ({ themes }) => {
    */
   const BUTTON_VARIANT_OVERRIDES_CONFIG = [
     {
-      name: 'Alert',
+      name: 'Alert actions',
```

Comment:
[inform] Opting to have a bit more granular/specific names vs just a global component name like `Alert`. That is, the names are now specific to their `selectors`.